File: atcoder/abc015_2.cpp
// https://atcoder.jp/contests/abc015/tasks/abc015_2
#include<iostream>
//#include<algorithm>
//#include<string>
#include<vector>
//#include<map>
//#include<tuple>
//#include<queue>
//#include<regex>

// #include <bits/stdc++.h>
using namespace std;

int main() {
	int N;
	cin >> N;
	vector<int> A(N);
	int sum = 0, cnt = 0;
	for (int i = 0; i < N; i++)
	{
		cin >> A[i];
		if (A[i] == 0) continue;
		else 
		{
			cnt++;
			sum += A[i];
		}
	}

	int ans = ceil((double)sum / cnt);
	cout << ans << endl;
	return 0;
}

File: atcoder/abc015_2.py
# https://atcoder.jp/contests/abc015/tasks/abc015_2

import math

N = int(input())
A = list(map(int, input().split()))

summ, cnt = 0, 0
for i in range(N):
    if A[i]  == 0:
        continue
    else:
        cnt += 1
        summ += A[i]

ans = math.ceil(summ / cnt)

print(ans)
